packages/strapi-admin/admin/src/translations/sv.json

```diff
@@ -0,0 +1,157 @@
+{
+  "Analytics": "Analytics",
+  "Content Manager": "Innehållshantering",
+  "Content Type Builder": "Innehållstyp Byggare",
+  "Email": "E-post",
+  "Files Upload": "Uppladdning av filer",
+  "HomePage.notification.newsLetter.success": "Prenumererade på nyhetsbrevet",
+  "New entry": "Nytt inlägg",
+  "Password": "Lösenord",
+  "Provider": "Leverantör",
+  "ResetPasswordToken": "Återställ lösenordstoken",
```

Comment:
Is this actually resetting the token, or initiating a reset password token? Because if it is the latter, "Återställ lösenord" should be the correct term.

Reply:
> Is this actually resetting the token, or initiating a reset password token? Because if it is the latter, "Återställ lösenord" should be the correct term.

Done.